xbmc/settings/Settings.cpp

```diff
@@ -63,7 +63,6 @@
 #include "peripherals/Peripherals.h"
 #include "powermanagement/PowerManager.h"
 #include "profiles/ProfilesManager.h"
-#include "pvr/PVRManager.h"
```

Comment:
You have still `  m_settingsManager->UnregisterCallback(&PVR::g_PVRManager);` on line 609.